build/templates/_converters.py.mako

```diff
@@ -7,6 +7,26 @@ from ${module_name} import visatype
 import datetime
 
 
+def convert_repeated_capabilities(repeated_capability, prefix=''):
+    # First try it as a list
+    if isinstance(repeated_capability, list):
```

Comment:
This might be too restrictive. What if the structure is something other than a list, like array.array? Python user would expect it to work because duck typing.
Consider implementing with try/except pattern instead of looking for specific type.

Reply:
I did that at first. This is fixing the issue that broke using string. Strings are iterable so it was iterating over the string and that is what would turn '0,1' into '0,,,1' and 'ScriptTrigger0' into 'S,c,r,i,p,t,T,r,i,g,g,e,r,0'.

I would need to explicitly look for a string and handle that first, and then try to iterate.

Reply:
I think that's reasonable.

Reply:
Changed the logic to first look for six.string_type or six.text_type. This allows us to accept any iterable.

This adds a dependency to six for all drivers.